omod-1.8/src/test/resources/create_patient_from_get.json

```diff
@@ -0,0 +1,267 @@
+                "address14":null,
+                "address15":null,
+                "links":[
+                    {
+                        "rel":"self",
+                        "uri":"http://192.168.1.100/openmrs/ws/rest/v1/person/a65fc361-f6ac-40ba-8485-94749c061509/address/ebd2ed47-6d13-41a0-bc6b-c1ce6b10da25"
+                    },
+                    {
+                        "rel":"full",
+                        "uri":"http://192.168.1.100/openmrs/ws/rest/v1/person/a65fc361-f6ac-40ba-8485-94749c061509/address/ebd2ed47-6d13-41a0-bc6b-c1ce6b10da25?v=full"
+                    }
+                ],
+                "resourceVersion":"2.0"
+            }
+        ],
+        "attributes":[
+            {
+                "display":"Chad",
+                "uuid":"800a5043-97b0-4ef8-9dfa-2075bce3a434",
+                "value":{
+                    "uuid":"e10ffe54-5184-4efe-8960-cd565ec1cdf8",
+                    "display":"Chad",
+                    "links":[
+                        {
+                            "rel":"self",
+                            "uri":"http://192.168.1.100/openmrs/ws/rest/v1/concept/e10ffe54-5184-4efe-8960-cd565ec1cdf8"
+                        }
+                    ]
+                },
+                "attributeType":{
+                    "uuid":"55e6ce9e-25bf-11e3-a013-3c0754156a5d",
+                    "display":"Caste",
+                    "links":[
+                        {
+                            "rel":"self",
+                            "uri":"http://192.168.1.100/openmrs/ws/rest/v1/personattributetype/55e6ce9e-25bf-11e3-a013-3c0754156a5d"
+                        }
+                    ]
+                },
+                "voided":false,
+                "links":[
+                    {
+                        "rel":"self",
+                        "uri":"http://192.168.1.100/openmrs/ws/rest/v1/person/a65fc361-f6ac-40ba-8485-94749c061509/attribute/800a5043-97b0-4ef8-9dfa-2075bce3a434"
+                    },
+                    {
+                        "rel":"full",
+                        "uri":"http://192.168.1.100/openmrs/ws/rest/v1/person/a65fc361-f6ac-40ba-8485-94749c061509/attribute/800a5043-97b0-4ef8-9dfa-2075bce3a434?v=full"
+                    }
+                ],
+                "resourceVersion":"1.8"
+            }
+        ],
+        "voided":false,
+        "auditInfo":{
+            "creator":{
+                "uuid":"1c3db49d-440a-11e6-a65c-00e04c680037",
+                "display":"admin",
+                "links":[
+                    {
+                        "rel":"self",
+                        "uri":"http://192.168.1.100/openmrs/ws/rest/v1/user/1c3db49d-440a-11e6-a65c-00e04c680037"
+                    }
+                ]
+            },
+            "dateCreated":"2018-08-26T10:40:30.000+0000",
+            "changedBy":null,
+            "dateChanged":null
+        },
+        "deathdateEstimated":false,
+        "birthtime":null,
```

Comment:
Same here, this was added in 1.10 so it should not be here, it's one of the reasons why the test fails since this file is in the 1.8 subproject and the test where it is used is a 1.9 test